framework/test/mocha/unit/modules/http_api/controllers/delegates.js

```diff
@@ -315,7 +315,7 @@ describe('delegates/api', () => {
 			try {
 				await __private.aggregateBlocksReward({ address: '1L' });
 			} catch (err) {
-				expect(err).to.deep.equal('Blocks#aggregateBlocksReward error');
+				expect(err.message).to.deep.equal('Blocks#aggregateBlocksReward error');
```

Comment:
🤔

Reply:
before it was not throwing the error, it was throwing the message, any specific reason that we throw the message and not the error with message?

Reply:
there was no specific message, but if we gonna change it it should be changed consistently in separate PR

Reply:
agree.. I will revert it back.. and create a separate issue to address this